app/scripts/components/dashboardComponents/listItems.jsx

```diff
@@ -136,6 +104,182 @@ class ListItems extends React.Component {
         );
     }
 
+    // tableHeaders
+    tableHeaderColumnCheckBox(showChecks, checkboxStyle, allItemsSelected) {
+        if (this.props.router.location.pathname !== '/dashboard' ) {
+            return (
+                <TableHeaderColumn style={styles.checkbox}>{showChecks && <Checkbox
+                    label="SELECT ALL"
```

Comment:
Need to disable "select all" for software agents

Reply:
I guess we also need to disable the upload button and add folder button too if on software agents page. I guess I never did that.

Reply:
Yea, It will need more work to add this implementation into Agents.
Mostly because Agents was not using ListItems to start with, but it should not be too much work. 🤞